models/uniswap/bnb/uniswap_bnb_pools.sql

```diff
@@ -0,0 +1,29 @@
+{{ config(
+    tags=['dunesql'],
+    schema = 'uniswap_bnb',
+    alias = 'pools',
```

Comment:
we need to ensure to use new alias macro on all config blocks. please use:
`alias('pools')`, similar to arbitrum version